addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,25 @@
+---Models in module 'website_livechat'---
+---Fields in module 'website_livechat'---
+website_livechat / mail.channel             / livechat_active (boolean)     : NEW
+website_livechat / mail.channel             / livechat_visitor_id (many2one): NEW relation: website.visitor
+website_livechat / website.visitor          / livechat_operator_id (many2one): NEW relation: res.partner, isfunction: function, stored
+website_livechat / website.visitor          / mail_channel_ids (one2many)   : NEW relation: mail.channel
+# Nothing to do
+---XML records in module 'website_livechat'---
+DEL im_livechat.channel: website_livechat.im_livechat_channel_data_website (noupdate)
```

Comment:
This is in fact renamed to `im_livechat.im_livechat_channel_data` (but in `im_livechat`), so it's interesting to mark it.

Reply:
I would say that `# NOTHING TO DO: Renamed to im_livechat.im_livechat_channel_data in im_livechat`

Reply:
Done.